ui/partials/perf/comparechooserctrl.html

```diff
@@ -10,7 +10,7 @@
           <label for="original-project-selector">Project</label>
           <select id="original-project-selector" class="form-control" ng-model="originalProject" ng-options="project.name for project in projects"/>
           <label for="original-revision-input">Revision</label>
-          <div class="form-group has-error">
+          <div class="form-group" ng-class="{'has-error' : originalRevisionError}">
```

Comment:
You added a space before and after the : where you should have only added one after the colon. Please change:

{'has-error' : originalRevisionError} -> {'has-error': originalRevisionError}